notes/bitcoin-as-anti-authoritarian.md

```diff
@@ -1,36 +1,50 @@
 ---
-title: Bitcoin as an Anti-Authoritarian Force
-date: 2022-05-02
-created: 2022-05-02
-description: "Evaluating the thesis that Bitcoin (and crypto more generally) is an anti-authoritarian force and can help undermine tyranny by providing a state-resistant payment rail."
-image: /img/anti-auth-thumbnail.png
+title: Crypto can provide a transnational state-resistant payment system for dissidents 
```

Comment:
I think old description was a bit pithier. I would keep structured description for the main part.

Reply:
i also think old title was nice and maybe could combine:

```
Bitcoin as an Anti-Authoritarian Force: Crypto can provide a transnational state-resistant payment system for dissidents 
```